lib/mix/tasks/school_house.gen.rss.ex

```diff
@@ -6,21 +6,31 @@ defmodule Mix.Tasks.SchoolHouse.Gen.Rss do
   """
 
   alias SchoolHouse.Posts
-  alias SchoolHouseWeb.{Endpoint, Router.Helpers}
+  alias SchoolHouseWeb.Router.Helpers
 
   @destination "priv/static/feed.xml"
 
-  def run(_args) do
-    Mix.Task.run("app.start")
+  def run([uri_string]) do
```

Comment:
These now accept a URI string when generating the static files. Removes the dependency on needed to start the app and have env values set.